stratify/_vinterp.pyx

```diff
@@ -544,10 +543,6 @@ cdef class _Interpolation(object):
         fz_src = fz_src.astype(np.float64)
 
         # Broadcast the z_target shape if it is 1d (which it is in most cases)
-        if z_target.ndim == 1:
-            z_target_size = z_target.shape[0]
-        else:
-            z_target_size = z_target.shape[axis]
```

Comment:
Gosh, these variable names throw me :)
Is it a good idea to have these descriptions as comments where/when the variable is defined in the code??